src/creature.h

```diff
@@ -186,6 +205,15 @@ class Creature
                                                    double total_dispersion );
 
         /**
+         * Makes an aiming/attack roll for a single projectile attack shot.
+         *
```

Comment:
You should still change it, if only as the start of some documentation of what you want.